apps/dashboard/app/javascript/packs/files/DataTable.js

```diff
@@ -0,0 +1,431 @@
+import 'datatables.net';
+import 'datatables.net-bs4/js/dataTables.bootstrap4';
+import 'datatables.net-select';
+import 'datatables.net-select-bs4';
+import Handlebars from 'handlebars';
+import {} from './SweetAlert.js';
+import {} from './FileOps.js';
+import {} from './UppyOps.js';
+import {} from './ClipBoard.js';
```

Comment:
Why does this file need these imports?

Reply:
Unfortunately, for the listener to get initialized, the files have to be loaded. Trust me, I was disappointed about that too. I figured to put them here and then when we move on to the filePicker, there can be flags that determine what needs to be loaded.

Reply:
OK - I'll look into it more. 

Which listener? With the exception of sweet alert, from what I can tell, you're just passing back strings. I'm not sure what exactly you're importing.

Reply:
I see you're using `DataTable.js` as the entry point? Yea I'm not sure if it's supposed to be the top level component.

Reply:
This has been migrated to index.js